### ISSUE TITLE
Having unsupported codes on oc-10 branch. 

### ISSUE THREAD

**Issue by parhamin2010** [pondersource/nc-sciencemesh#260]
we have unsupported codes come from oc copy. 
We have PR to main branch and this should be fixed after this PR fully reviewed
https://github.com/pondersource/nc-sciencemesh/pull/256

**Comment by michielbdejong** [comment]
What does this mean? What are unsupported codes?

**Comment by parhamin2010** [comment]
I created another branch called oc-10-fixes and created a PR with oc-10.
We've some commits on oc-10 that will fix those unsupported codes. 
The reason I create this branch is about checking the app is running properly on NC/OC with those commits after oc-10 merge with a similiar branch to main like oc-10-fixes and then we can merge oc-10-fixes to main to finalize the deprication of OC repo.

**Comment by michielbdejong** [comment]
That's still not a good description for a GitHub issue. Can you describe what the problem is in terms of:
* steps to reproduce
* expected behaviour
* actual behaviour
* any further comments, for instance if you think you know what may be causing the problem

**Comment by parhamin2010** [comment]
- steps to reproduce:
1- I created a new branch from main called oc-10-fixes
2- Create a new PR that merged oc-10 to oc-10-fixes 
3- Check oc-10-fixes on dev-stock to find any conflict/issues/bugs

- expected behaviour:
Similar behaviour as we had on main branch on NC and OC repo

- actual behaviour
It was showing error that saying that (some codes are unsupported by the developed app) on main screen once we run nrrn script. I didn't get any screenshot of it.
Once I refresh the page sciencemesh app disabled and once I tried multiple times to enable/active it, it didn't work.
I checked the logs and found out this was the root cause.

Fixed in this PR:
https://github.com/sciencemesh/nc-sciencemesh/commit/665fbbcc8b5f2e3c37d171303f3e0a16437e3969
